doc/app/ArticlesController.html

```diff
@@ -188,7 +188,7 @@ <h3>Public Instance Methods</h3>
 <span class="ruby-keyword">def</span> <span class="ruby-identifier">destroy</span>
   <span class="ruby-ivar">@article</span>.<span class="ruby-identifier">destroy</span>
   <span class="ruby-identifier">flash</span>[<span class="ruby-value">:success</span>] = <span class="ruby-node">&quot;Article was removed! #{make_undo_link}&quot;</span>
-  <span class="ruby-constant">UserNotifier</span>.<span class="ruby-identifier">notify_of_removal</span>(<span class="ruby-ivar">@article</span>.<span class="ruby-identifier">followers</span>,<span class="ruby-ivar">@article</span>).<span class="ruby-identifier">deliver_now</span>
+  <span class="ruby-constant">UserMailer</span>.<span class="ruby-identifier">notify_of_removal</span>(<span class="ruby-ivar">@article</span>.<span class="ruby-identifier">followers</span>,<span class="ruby-ivar">@article</span>).<span class="ruby-identifier">deliver_now</span>
```

Comment:
You didn't do these changes manually, did you?  Anything in the `doc` directory can be ignored, honestly.

Reply:
I actually do these manually .. so I need to undo all the changes in the `doc` directory ?

Reply:
Oh no, you can leave them there.  I was just saying that you didn't have to do them.